optimade/models/tests/test_models.py

```diff
@@ -1,14 +1,14 @@
 import pytest
 from pydantic import BaseModel, ValidationError, ConfigError
 
-from .util import conlist
+from optimade.models.util import conlist
```

Comment:
This is made redundant by all the validators in this PR, but still would be good to include these constraints in the generated spec somehow.